rust/datafusion/src/physical_plan/hash_aggregate.rs

```diff
@@ -734,16 +734,10 @@ fn aggregate_batch(
 
             // 1.3
             match mode {
-                AggregateMode::Partial => {
-                    accum.update_batch(values)?;
-                }
-                AggregateMode::Final => {
-                    accum.merge_batch(values)?;
-                }
+                AggregateMode::Partial => accum.update_batch(values),
+                AggregateMode::Final => accum.merge_batch(values),
             }
-            Ok(accum)
         })
-        .collect::<Result<Vec<_>>>()
```

Comment:
Small cleanup here, to not collect it needlessly into a vec (as it is the same as the input anyway, and it mutates the accumulators, so it's more clear and slightly more efficient now).

Reply:
makes sense to me. 👍